source/iostreams.tex

```diff
@@ -6029,7 +6114,7 @@
 defines a class that is responsible for doing exception safe prefix and suffix
 operations.
 
-\indexlibrary{\idxcode{sentry}!constructor}%
```

Comment:
As far as I can tell, I am consistent for constructor and destructor - the difference is between basic_istream::sentry and basic_ostream::sentry.  In one case, I retain the old index entry and add the new one, in the other, I just rename the old to the new.

I agree they should be consistent, but not sure which way we think is the best to resolve.  I mildly lean towards retaining both 'sentry!constructor' and 'basic_Xstream::sentry!constructor' (and similarly for destructors), but I lean the other way for 'operator bool' - with is terribly inconsistent of me again.

Preferences?